ipwb/indexer.py

```diff
@@ -224,6 +226,16 @@ def getCDXJLinesFromFile(warcPath, **encCompOpts):
             entry.buffer.seek(0)
             payload = entry.buffer.read()
 
+            title = None
+            try:
+                ctype = hdrs.get_header('Content-Type')
+                if (ctype and ctype.lower().startswith('text/html')):
```

Comment:
Are the outer parens necessary? We don't use that form elsewhere.

Reply:
No, these are not necessary. Initially, I was using header getter inline and did not have that `ctype` variable, so the conditional line was exceeding the pycodestyle limits. In that case generally parens are placed and lines are broken. later, I extracted the repeated part in a variable and it was not needed anymore, but forgot to remove it. Will send commit soon to fix it.